packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -282,6 +282,17 @@ export type VideoSource =
        * @platform ios
        */
       headers?: Record<string, string>;
+
+      /**
+       * @experimental
+       * Specifies whether the player should use caching for the video.
+       * > Due to platform limitations caching HLS video sources on iOS is not supported.
+       * > Caching DRM-protected videos is not supported.
```

Comment:
```suggestion
       * > Caching DRM-protected videos is not supported on Android and iOS.
```

Based on the comment and context I left in the Expo Video API reference page earlier.